admin_guide/managing_projects.adoc

```diff
@@ -376,3 +376,48 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 endif::[]
+
+ifdef::openshift-dedicated[]
+[[enforcing-different-limits-on-users-projects]]
+== Enforcing Different Limits on Users' Projects
+
+If you are an {product-title} administrator and want to enforce different limits
+on the number of projects that your users can create, apply the appropriate
+label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
```

Comment:
This admission controller is already configured for dedicated clusters. No need to specify this configuration.